include/bluetooth/hci.h

```diff
@@ -1335,6 +1335,28 @@ struct bt_hci_cp_le_set_privacy_mode {
 	uint8_t         mode;
 } __packed;
 
+#define BT_HCI_LE_1US_SWITCHING_FOR_AOD_TX      BIT(0)
+#define BT_HCI_LE_1US_SAMPLING_FOR_AOD_RX       BIT(1)
+#define BT_HCI_LE_1US_SAMPL_SWITCH_FOR_AOA_RX   BIT(2)
```

Comment:
Use consistent prefix. I have not read the spec., but I am guessing there are options for antennae switching/sampling

Reply:
Actually this one is inconsistent... Those defines are related with non-mandatory modes for AoD and AoA:

From BT Spec Vol 4: 7.8.87
Supported_Switching_Sampling_Rates:
- bit nr: 0 "1 μs switching supported for AoD transmission"
- bit nr: 1 "1 μs sampling supported for AoD reception"
- bit nr: 2 "1 μs switching and sampling supported for AoA reception"
- bit nr: 3-7 "Reserved for future use"

2μs switching and sampling is mandatory.

Reply:
What do you think if I use following names?

#define BT_HCI_LE_1US_AOD_TX	BIT(0)
#define BT_HCI_LE_1US_AOD_RX	BIT(1)
#define BT_HCI_LE_1US_AOA_RX	BIT(2)

They are more consistent. Also AOD_TX is implicitly related with switching and AOD_RX is implicitly related with sampling so there is no need to put it in names.

Reply:
Changed to last proposed by me version